test/asyncapiSchemaFormatParser_test.js

```diff
@@ -47,18 +47,6 @@ describe('asyncapiSchemaFormatParser', function() {
           endOffset: offset(297, 15)
         }
       },
-      {
-        title: '/channels/mychannel/publish/message/payload/additionalProperties should be object',
-        location: {
-          jsonPointer: '/channels/mychannel/publish/message/payload/additionalProperties',
-          startLine: 13,
-          startColumn: 38,
-          startOffset: offset(252, 13),
-          endLine: 15,
-          endColumn: 15,
-          endOffset: offset(297, 15)
-        }
-      },
```

Comment:
When we will merge it https://github.com/asyncapi/asyncapi-node/pull/63 then this error won't be throw.